backend/module/authn/authn.go

```diff
@@ -55,6 +57,33 @@ type api struct {
 }
 
 func (a *api) Login(ctx context.Context, request *authnv1.LoginRequest) (*authnv1.LoginResponse, error) {
+	// Attempt refresh. If refresh fails for any reason continue the regular auth flow.
+	md, _ := metadata.FromIncomingContext(ctx)
+	if v := md.Get("grpcgateway-cookie"); len(v) > 0 {
+		if refreshToken, err := mux.GetCookieValue(v, "refreshToken"); err == nil {
+			if t, err := a.issuer.RefreshToken(ctx, &oauth2.Token{RefreshToken: refreshToken}); err == nil {
```

Comment:
This refresh error does not get surfaced is this on purpose?

Reply:
Asking because OIDC has some refresh token policy errors (like `obsolete cause its used twice`, `expiration`,etc ) that might be helpful to surface

Reply:
yep, the user should fall back to the regular auth flow since their refresh token is invalid. however i can split this out into another method and then log any error for debug purposes. will do that.

Reply:
👍  logging the error would be helpful to identify a scenario where the refresh never happens due so some policy error